server-common/src/main/java/org/apache/kafka/server/share/persister/PersisterStateManager.java

```diff
@@ -437,6 +437,10 @@ protected void handleFindCoordinatorResponse(ClientResponse response) {
 
             FindCoordinatorResponseData.Coordinator coordinatorData = coordinators.get(0);
             Errors error = Errors.forCode(coordinatorData.errorCode());
+            String errMessage = coordinatorData.errorMessage();
```

Comment:
nit: `errorMessage` has been used in the other cases.

Reply:
I see you fixed this in the next commit.

Reply:
State file